Trying to generate the website is a bit obscure and hard

### Description

It is using `npm`, rather than `make`. And the old version of hugo used seems a bit brittle, as well.

I was trying to preview a small change that I did, and it was harder than what it should have been.

Ended up at https://gohugo.io/ and scrolling through package.json.

Not sure what was up with hugo_extended_0.113.0_linux-amd64.deb

```
panic: interface conversion: interface {} is bool, not maps.Params

goroutine 1 [running]:
github.com/gohugoio/hugo/config.(*defaultConfigProvider).Merge(0xc000476a80, {0x0, 0x0}, {0x223c640, 0xc000d7faa0?})
	/root/project/hugo/config/defaultConfigProvider.go:223 +0x874
github.com/gohugoio/hugo/config/allconfig.(*configLoader).loadModules.func1(0xc00012e320?)
	/root/project/hugo/config/allconfig/load.go:444 +0x20d
github.com/gohugoio/hugo/modules.(*Client).Collect(0xc000493600)
	/root/project/hugo/modules/collect.go:66 +0x189
github.com/gohugoio/hugo/config/allconfig.(*configLoader).loadModules(0xc00020bcc0, 0x23749b9?)
	/root/project/hugo/config/allconfig/load.go:465 +0x425
github.com/gohugoio/hugo/config/allconfig.LoadConfig({{0x2a411a0, 0x3c261a0}, {0x2a44630, 0xc000317c20}, {0x2a420f0, 0xc000476a40}, {0x0, 0x0}, {0x23735c7, 0x6}, ...})
	/root/project/hugo/config/allconfig/load.go:66 +0x585
github.com/gohugoio/hugo/commands.(*rootCommand).ConfigFromProvider.func1(0x317cb8?)
	/root/project/hugo/commands/commandeer.go:220 +0x358
github.com/bep/lazycache.(*Cache[...]).GetOrCreate(0x2a53360, 0x0, 0xc000dafa80?)
	/root/project/gomodcache/github.com/bep/lazycache@v0.2.0/lazycache.go:112 +0x142
github.com/gohugoio/hugo/commands.(*rootCommand).ConfigFromProvider(0xc0000552c0?, 0x2a420f0?, {0x2a420f0, 0xc000476a40})
	/root/project/hugo/commands/commandeer.go:196 +0x68
github.com/gohugoio/hugo/commands.(*hugoBuilder).loadConfig(0xc000263280, 0x6?, 0x1)
	/root/project/hugo/commands/hugobuilder.go:1017 +0x3ac
github.com/gohugoio/hugo/commands.(*serverCommand).PreRun(0xc000ae6960, 0xc0000552c0, 0xc0000552c0?)
	/root/project/hugo/commands/server.go:607 +0x24a
github.com/bep/simplecobra.(*Commandeer).init(0xc0000552c0)
	/root/project/gomodcache/github.com/bep/simplecobra@v0.3.1/simplecobra.go:92 +0xcb
github.com/bep/simplecobra.(*Commandeer).compile.func2(0xc000afa600?, {0x236a265?, 0x0?, 0x0?})
	/root/project/gomodcache/github.com/bep/simplecobra@v0.3.1/simplecobra.go:119 +0x1d
github.com/spf13/cobra.(*Command).execute(0xc000afa600, {0x3c26820, 0x0, 0x0})
	/root/project/gomodcache/github.com/spf13/cobra@v1.7.0/command.go:925 +0x7db
github.com/spf13/cobra.(*Command).ExecuteC(0xc000005800)
	/root/project/gomodcache/github.com/spf13/cobra@v1.7.0/command.go:1068 +0x3bd
github.com/spf13/cobra.(*Command).ExecuteContextC(...)
	/root/project/gomodcache/github.com/spf13/cobra@v1.7.0/command.go:1001
github.com/bep/simplecobra.(*Exec).Execute(0xc0005a8510, {0x2a35bc8?, 0xc000130000?}, {0xc00012c010?, 0x509b88?, 0xc000100568?})
	/root/project/gomodcache/github.com/bep/simplecobra@v0.3.1/simplecobra.go:155 +0xd9
github.com/gohugoio/hugo/commands.Execute({0xc00012c010, 0x1, 0x1})
	/root/project/hugo/commands/commandeer.go:66 +0x312
main.main()
	/root/project/hugo/main.go:25 +0x65
```

But ended up patching hugo, to get it running (`make -C .. docsy && npm run serve`)

There should be probably be some documentation in website, on how to build it?

--- a/website/Makefile
+++ b/website/Makefile
@@ -1,0 +1,12 @@
+# See also: package.json
+NPM ?= npm
+
+build serve: _output/docsy
+	$(NPM) run $@
+
+_output/docsy:
+	$(MAKE) -C .. docsy
+
+clean:
+	$(NPM) run $@
+	rm -Rf _output

